Readme.md

```diff
@@ -623,6 +623,17 @@ connection.query('SELECT * FROM users WHERE id = ?', [userId], function(err, res
 });
 ```
 
+Multiple placeholders are mapped to values in the same order as passed. For example:
+
+```js
+var values = ['a', 'b', 'c']
+connection.query('INSERT INTO users(foo, bar, baz) VALUES(?, ?, ?)', [userId], function(err, results) {
```

Comment:
Thanks @dougwilson. That was a complete copy/paste error. I've fixed it and made a slight adjustment for hopefully better clarity.